frontend/src/locale/index.ts

```diff
@@ -1913,6 +1968,57 @@ Affrettati a comprare il biglietto!`,
     "profile.myProfile": "Il mio profilo",
     "profile.myProposals": "Le mie proposte",
 
+    "profile.myGrant": "Il mio grant",
+    "profile.myGrant.nextSteps": "I prossimi passi",
+    "profile.myGrant.grantType": "Tipo di Grant",
+    "profile.myGrant.appliedFor": "Richiesto",
+    "profile.myGrant.appliedFor.ticket": "Biglietto",
+    "profile.myGrant.appliedFor.travel": "Viaggio",
+    "profile.myGrant.appliedFor.accommodation": "Alloggio",
+    "profile.myGrant.edit": "Modifica",
+    "profile.myGrant.editInfo":
+      "Assicurati che tutte le informazioni fornite siano corrette. Hai tempo fino al {editDeadline} per modificare il tuo grant.",
+    "profile.myGrant.manage": "Gestisci",
+    "profile.myGrant.manage.warning": `Se i tuoi piani cambiano o non puoi più partecipare alla conferenza, per favore faccelo sapere cliccando sul pulsante 'Gestisci' per rifiutare il tuo grant o inviaci un'e-mail a {grantsEmail} il prima possibile.
+La mancata comunicazione potrebbe influenzare la tua idoneità a ricevere aiuti finanziari per eventi futuri.`,
+    "profile.myGrant.status": "Stato",
+    "profile.myGrant.status.pending": "In attesa",
+    "profile.myGrant.status.pending.nextSteps":
+      "La tua domanda è in fase di revisione. Non è richiesta alcuna azione in questo momento.",
+    "profile.myGrant.status.rejected": "Respinto",
+    "profile.myGrant.status.rejected.nextSteps":
+      "Purtroppo, a causa dei fondi limitati e del grande numero di richieste, la tua richiesta di grant non è stata approvata. Controlla la tua email per maggiori dettagli.",
+    "profile.myGrant.status.approved": "Approvato",
+    "profile.myGrant.status.approved.nextSteps":
+      "La tua richiesta di grant è stata approvata. Ti invieremo un'email molto presto con ulteriori istruzioni. Per favore, controlla la tua email per queste istruzioni.",
+    "profile.myGrant.status.waiting_list": "Lista d'attesa",
+    "profile.myGrant.status.waiting_list.nextSteps":
+      "Sei in lista d'attesa. Ti informeremo non appena diventano disponibili ulteriori fondi o se qualcuno rifiuta il suo grant.",
```

Comment:
```suggestion
      "Sei in lista d'attesa. Ti informeremo non appena potremo accettare il tuo grant.",
```
I felt we were giving too much detail